src/js/profile/mobile/widget/DropdownMenu.js

```diff
@@ -733,7 +733,12 @@
 					selectors.getParentsByClass(element, pageClasses.uiHeader)[0];
 				ui.elDefaultOption = element.querySelector("option[data-placeholder='true']");
 
-				self._selectedIndex = element.selectedIndex;
+				// check if data placeholder exist on index 0
+				if (ui.elDefaultOption && ui.elDefaultOption.index == 0 && element.selectedIndex > 0) {
```

Comment:
Wondering if we need to assume that placeholder exists on index 0. This obviously happens in our example but maybe we should allow developer to set placeholder at any option he want. What do you think?

Reply:
you are right, I improved code